Frontend/src/api/helper/backend_fetch.ts

```diff
@@ -0,0 +1,34 @@
+type Method = 'POST' | 'GET' | 'PATCH' | 'DELETE'
+
+type Body =
```

Comment:
I don't think this will scale. Rather backendFetch should take any in as Body and each function that uses backendFetch should define it's own body props.

Reply:
Or TS also likes using `unknown` in these situations as opposed to `any`

Reply:
Yes, this is not meant to scale. All these types will be generated automatically from our API Spec